cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerLoggingConfigurationDiff.java

```diff
@@ -179,4 +111,45 @@ private static boolean isValidLoggerLevel(String level) {
         return VALID_LOGGER_LEVELS.contains(level);
     }
 
+    static class LoggingLevelResolver {
+
+        Map<String, String> config;
+
+        LoggingLevelResolver(Map<String, String> loggingConfig) {
+            this.config = loggingConfig;
+        }
+
+        LoggingLevel resolveLevel(String name) {
+            String level = config.get(name);
+            if (level != null) {
+                return LoggingLevel.valueOf(level.split(",")[0]);
+            }
+
+            int e = name.length();
```

Comment:
Can you put there any doc?

Reply:
Not sure if you have something specific in mind for this section of the code, or do you mean more docs generally? Any specific questions you are looking clarifications for?